component/Page/Profile/index.tsx

```diff
@@ -154,7 +155,9 @@ export default function ProfilePage(props: PageProps) {
             </Helmet>
             <LayoutContainer backgroundImage="/image/background-profile.svg">
                 <Header fixedPosition={false}/>
-                <div className="flex flex-1-0-0 w-full justify-center z-50">
+                <div className="flex flex-1-0-0 w-full justify-center" style={{
+                    zIndex: 1000000
+                }}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `zIndex` value seems excessively high. Unless there's a specific reason for this, it might be better to use a lower value. High z-index values can lead to stacking context issues and make the CSS harder to manage.
```diff
- zIndex: 1000000
+ zIndex: 50
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->